packages/dds/tree/src/test/edit-manager/editManager.spec.ts

```diff
@@ -216,42 +226,42 @@ function editManagerFactory(): {
 } {
     const { rebaser, family } = changeFamilyFactory();
     const manager = new EditManager<TestChangeset, ChangeFamily<unknown, TestChangeset>>(
-        localSessionId,
         family,
     );
+    manager.setLocalSessionId(localSessionId);
     return { rebaser, manager };
 }
 
-const localSessionId: SessionId = brand(0);
-const peerSessionId1: SessionId = brand(1);
-const peerSessionId2: SessionId = brand(2);
+const localSessionId: SessionId = "0";
+const peerSessionId1: SessionId = "1";
+const peerSessionId2: SessionId = "2";
 
 const NUM_STEPS = 5;
 const NUM_CLIENTS = 3;
 
 describe("EditManager", () => {
-    it("Can handle non-concurrent local changes being sequenced immediately", () => {
+    it("Test#1", () => {
```

Comment:
You didn't like the test name? :)

Reply:
Oops. I was having trouble getting the tests to rebuild, and I renamed one of them to quickly see if my changes were being reflected in the test run.